.circleci/config.yml

```diff
@@ -1,164 +1,113 @@
 version: 2.1
 
-parameters:
-  preview:
-    type: boolean
-    default: true
-  3-10:
-    type: boolean
-    default: true
-  3-11:
-    type: boolean
-    default: true
+# orbs:
+#   coveralls: coveralls/coveralls@2.2.1
+
+workflows:
+  ci:
+    jobs:
+      - lint
+      - test:
+          matrix:
+            parameters:
+              # TODO: Revisit why pyenv doesn't recognize 3.12
+              python_version: ["3.8", "3.9", "3.10", "3.11"] # "3.12"
+              arangodb_config: ["single", "cluster"]
+              arangodb_license: ["community", "enterprise"]
```

Comment:
Want thing I just want to clarify according to the current state of `starter.sh`

`community` --> `docker/arangodb` (https://hub.docker.com/r/arangodb/arangodb/tags)
`enterprise` --> `docker/enterprise` (https://hub.docker.com/r/arangodb/enterprise/tags)

Perhaps we can revisit the idea of `community-preview` (https://hub.docker.com/r/arangodb/arangodb-preview/tags) and `enterprise-preview` (https://hub.docker.com/r/arangodb/enterprise-preview/tags)